package.json

```diff
@@ -4,18 +4,21 @@
   "private": true,
   "dependencies": {
     "@giscus/react": "^2.0.3",
+    "@nhost/react": "^0.9.0",
     "@types/lodash": "^4.14.182",
     "@types/react": "^18.0.6",
     "@types/react-dom": "^18.0.2",
     "date-fns": "^2.28.0",
     "firebase": "^9.8.2",
+    "graphql": "^16.5.0",
     "lodash": "^4.17.21",
     "node-sass": "^7.0.1",
     "plop": "^3.0.5",
     "react": "^18.0.0",
     "react-dom": "^18.0.0",
     "react-helmet": "^6.1.0",
     "react-icons": "^4.3.1",
+    "react-loader-spinner": "^5.1.5",
```

Comment:
Has it been discussed to use this spinner?

Reply:
@koustov Yes, we have discussed and taken a call on this. This library provides plenty of great loading indicator suggestions to pick from and was used before. Right now we are using the rotating clock to indicate the data is loading but we can use anything else in future for the same. Also the indicator is highly customizable with color and all.